setup.py

```diff
@@ -38,7 +38,7 @@ def load_about():
     long_description=load_readme(),
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```

Comment:
I think [this doc](https://github.com/eduNEXT/drydock/blob/main/docs/decisions/001-Purpose.md) can be removed in this release since it no longer matches the rationale of this plugin